templates/default/deploy-scripts/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+edition = "2021"
+name = "deploy_scripts"
+version = "1.0.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+chrono = "0.4.26"
+tokio = {version = "1.18", features = ["rt", "macros", "rt-multi-thread"] }
+clap = { version = "4", features = ["derive", "env"]}
+concordium-rust-sdk="2.4"
```

Comment:
I just noticed this, the latest version of the Rust SDK is 3, which supports P6. We need to use that.